spec/features/read_story_spec.rb

```diff
@@ -59,4 +59,44 @@
       expect(page).to have_selector("span.merge")
     end
   end
+
+  feature "reading saved stories" do
+    let(:user) { create(:user) }
+
+    before do
+      story.update!(user:, editor: user)
```

Comment:
Please edit these tests to create new stories in the correct state instead of editing., I find it really hard to follow as the test file gets longer because I have to look up through each `before` and replay them in my head.

So eg `let(:user_edited_story) { create(:story, editor: user, [is_deleted: true]) }` and so on